libr/core/cmd_print.c

```diff
@@ -2235,6 +2235,7 @@ static void disasm_strings(RCore *core, const char *input, RAnalFunction *fcn) {
 	const char *linecolor = NULL;
 	char *ox, *qo, *string = NULL;
 	char *line, *s, *str, *string2 = NULL;
+	char *switchcmp = malloc (core->blocksize); 
```

Comment:
thats never freed. so its leaking.

you should set this var to NULL by default